docs/api.md

```diff
@@ -1703,9 +1703,12 @@ you are opening.
 #### bot.openVillager(villagerEntity)
 
 Returns an `Villager` instance which represents the trading window you are opening.
+You can listen to the `ready` event on this `Villager` to know when it's ready
 
 #### bot.trade(villagerInstance, tradeIndex, [times], [cb])
 
+This function also returns a `Promise`, with `void` as its argument upon completion.
```

Comment:
There are many javascript developers that are unfamiliar with void concept.

Reply:
And it's not an argument, rather a resolved value.

Reply:
I just mean that it's a void promise, there is no argument type

Reply:
All the other promise functions also contain this line